src/pages/[slug].astro

```diff
@@ -9,7 +29,7 @@ import Venue from "../components/Venue.astro";
 import HotelList from "../components/HotelList.astro";
 import RestaurantList from "../components/RestaurantList.astro";
 import AirtableForm from "../components/AirtableForm.astro";
-
+console.log(Astro.params);
```

Comment:
**suggestion**: guess this can be removed

Reply:
Done